sql/moz-fx-data-marketing-prod/acoustic_external/dataset_metadata.yaml

```diff
@@ -0,0 +1,11 @@
+friendly_name: Acoustic External
+description: |-
+  One time data dump of tables from Acoustic.
+  As part of the migration to Braze, all the data from Acoustic will be exported and uploaded to BigQuery once.
+dataset_base_acl: restricted
+user_facing: false
+labels: {}
+workgroup_access:
```

Comment:
Per https://mozilla-hub.atlassian.net/browse/DENG-3168?focusedCommentId=851334 as this dataset is going to likely replace `moz-fx-data-marketing.acoustic` we should consider adding some `roles/bigquery.dataEditor` access here. If we just need to do a one-time copy of the current dataset contents I don't think we should add it, but if DE are going to need to add more stuff manually we will need to codify that here.

Reply:
A quick question @whd related to this config and permissions management. This configuration file is for defining the dataset inside the `moz-fx-data-markering-prod` BigQuery project. Are the `dataset_metadata.yaml` files also used to manage this project via the SRE automation?